source/javascripts/components/ProductTour/HighlighterProductTooltip.tsx

```diff
@@ -105,7 +105,7 @@ export const HighlighterProductTooltip = ({
                     </Box>
 
                 {finished ? (
-                    <Box>
+                    <Box display="flex" flexDirection="column" gap="8">
```

Comment:
Can you add a screenshot or something for this? wondering if `flexDirection="column"` is what we want?